js/core/webidl-contiguous.js

```diff
@@ -640,9 +640,12 @@ define(
 
                     case "enum":
                         name = defn.name;
-                        defn.values.filter(function (v) { return v.type === undefined;})
-                                   .forEach(function(v) {
-                            v.dfn = findDfn(name, v, definitionMap, msg);
+                        defn.values.forEach(function(v,i) {
+                            if (v.type === undefined) {
```

Comment:
we need to keep the items that have .type different from undefined in defn.values; filtering them out was what was causing the previous Travis failure.

Given your note above, I'll merge this PR; hopefully someone can make a release out of it.